code/game/machinery/computer/communications.dm

```diff
@@ -411,15 +411,35 @@ GLOBAL_VAR_INIT(cops_arrived, FALSE)
 					to_chat(usr, span_warning("You are not authorized to do this!"))
 					playsound(src, 'sound/machines/terminal_prompt_deny.ogg', 50, FALSE)
 					return
+			else
+				to_chat(usr, "The console refuses to let you dial 911 as an AI or Cyborg!")
+				return
+			message_admins("[ADMIN_LOOKUPFLW(usr)] is considering calling the Sol Federation Marshals.")
+			if(tgui_input_list(usr, "Are you sure you want to call 911? Faulty 911 calls results in a $20,000 fine and a 5 year superjail \
+			sentence.", "Call 911", list("Yes", "No")) != "Yes")
+				return
+			message_admins("[ADMIN_LOOKUPFLW(usr)] has acknowledged the faulty 911 call consequences.")
+			if(tgui_input_list(usr, "You SHOULD call Marshals for:\n\
+			Security ignoring Command, Security violating civil rights, Security engaging in Mutiny, General Violation of Sol Federation Citizen Rights by Command/Security, etc.\n\
+			You SHOULD NOT call Marshals for:\n\
+			Corporate affairs, antagonist hunting, settling arguments, etc.\n\
```

Comment:
I wouldn't literally say "Antagonist hunting" lmao

Reply:
I don't want to leave any ambiguity for people on that one.

Reply:
I mean, it can be said differently. I'm sure we can come up with something better.